components/GetstartedCard2/index.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames'
+import Styles from './index.module.scss'
+import {
+    startPaymentData
+   
+} from './data'
+import LzLink from 'components/UI/link'
+import LinkIcon from 'public/icons/link-icon'
+import ArrowRightIcon from 'public/icons/arrow-right-icon'
+
+interface MyProps {
+    desc: string,
+    title: string,
+    route: string,
+}
+
+// card with Icon component
```

Comment:
There's no need to create two components, one with an icon and one without. Please use one component and conditionally render the icon if an icon prop is passed.

Reply:
fixed